docs/versioning.md

```diff
@@ -2,33 +2,22 @@
 
 ## Introduction
 
-Fedora 4 provides support for making snapshots of the current state of a resource which in turn become part of the resources version history.  The RDF for historic version snapshots may be browsed and old non-RDF content may be downloaded.  Furthermore, an object or subgraph may be reverted to the state that it existed in a historic version.
-
-The Fedora API Specification supports all the versioning capabilities for Fedora 4, the api for accessing these features will completely change.  To wit, the Fedora API specification conforms to the [Memento](https://mementoweb.org/guide/rfc/) specification.  Memento brings with capabilities that were not available in Fedora 4.  Below we will explore the most vital of these features as well as compare Fedora 4 with the Memento way of doing versioning.
+The Fedora API specification conforms to the [Memento](https://mementoweb.org/guide/rfc/) specification.  Memento brings with it all the versioning capabilities of Fedora 4 along with some new features. Below we will explore the most vital of these new features as well as compare the Fedora 4 with the Memento way of doing versioning.
 
 ## Overview of the New Memento Hotness
 
-The Memento spec employs some terminology which you may find a bit opaque.  You may find [this diagram](http://mementoweb.org/guide/quick-intro/) helpful in conceptualizing how memento works.  One of the key concepts to grasp is the notion of `TimeGates`.  A `TimeGate` is simply a resource that, when given a date time, will resolve a memento that most closely represents the state of the `resource` on that date.   Instead of having to know the exact date of the memento you wish to retrieve, the `TimeGate` essentially allows you to query the set of mementos for the closest match.  Furthermore the `TimeGate` will return the available time range,  the closest memento, the first and last mementos in the set, and the next and previous mementos relative to the closest match.  So if you wish to move forward or backward in time, you have the information you need.
+The Memento specification makes use of a handful of essential concepts.  You may find [this diagram](http://mementoweb.org/guide/quick-intro/) helpful in gaining an overall understanding of how Memento works.  One of the key concepts to grasp is the notion of `TimeGates`.  A `TimeGate` is simply a resource that, when given a date time, will resolve a memento that most closely represents the state of the `resource` on that date.   Instead of having to know the exact date of the memento you wish to retrieve, the `TimeGate` essentially allows you to query the set of mementos for the closest match.  Furthermore the `TimeGate` will return the available time range,  the closest memento, the first and last mementos in the set, and the next and previous mementos relative to the closest match.  So if you wish to move forward or backward in time, you have the information you need.
```

Comment:
A `TimeGate` _may_ return the closest memento, first/last, etc. But it is also entirely possible that the `TimeGate` will _not_ return that information but will simply respond with the Memento itself. Again, see the various interaction models that are possible.